src/AppBundle/Form/Pa/ProfileType.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace AppBundle\Form\Pa;
+
+use AppBundle\Entity\User;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+/**
+ * Class ProfileType
+ **
+ * @package AppBundle\Form\Pa
+ */
+class ProfileType extends AbstractType
+{
+    /**
+     * @var User
+     */
+    private $loggedInUser = null;
+
+    /**
+     * @param $loggedInUser
+     */
+    public function __construct(User $loggedInUser)
+    {
+        $this->loggedInUser = $loggedInUser;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('firstname', 'text', ['required' => true])
+            ->add('lastname' , 'text', ['required' => true])
+            ->add('email'    , 'text', ['required' => true])
+            ->add('jobTitle' , 'text', ['required' => !empty($this->loggedInUser)])
+            ->add('phoneMain', 'text', ['required' => !empty($this->loggedInUser)]);
```

Comment:
Minor/non-blocking thing: fields until here are the same as TeamMemberAccountType.
Do you think it'd be worth sharing that by inheriting the forms?